Cargo.toml

```diff
@@ -21,28 +22,29 @@ bindgen = "0.68.1"
 cc = "1.0.83"
 clap = { version = "4.2.1", features = ["derive", "env"] }
 clap_complete = "4.2"
+color-print = "0.3.5"
 comfy-table = { version = "7.0", features = ["custom_styling"] }
 directories = "5.0"
 dotenvy = "0.15"
 futures = "0.3"
 indexmap = { version = "2.0", features = ["serde"] }
-k8s-openapi = { version = "0.19", default-features = false, features = ["v1_27"] }
-kube = { version = "0.85", default-features = false, features = ["client", "rustls-tls"] }
+k8s-openapi = { version = "0.20", default-features = false, features = ["v1_28"] }
+kube = { version = "0.86", default-features = false, features = ["client", "rustls-tls"] }
 lazy_static = "1.4"
-nu-ansi-term = "0.49"
 once_cell = "1.18"
 phf = "0.11"
 phf_codegen = "0.11"
 rand = "0.8"
 regex = "1.9"
 reqwest = { version = "0.11", default-features = false, features = ["rustls-tls"] }
-semver = "1.0"
+semver = { version = "1.0", features = ["serde"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_yaml = "0.9"
 sha2 = "0.10"
 snafu = "0.7"
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.47.0" }
+spinoff = "0.8.0"
```

Comment:
Not an issue per se, but [indicatif](https://docs.rs/indicatif/latest/indicatif/) has pretty nice support for more detailed progress bars, and displaying multiple progress bars at once.

Reply:
Yes, agreed. I will start looking into this with the upcoming PR, which improves on the progress reporting.